<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>正则表达式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>NFA-&gt;DFA</string>
          </property>
         </widget>
        </item>
        <item>
          <widget class="QTableWidget" name="tableWidget_dfa">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>正则表达式-&gt;NFA</string>
          </property>
         </widget>
        </item>
        <item>
          <widget class="QTableWidget" name="tableWidget_nfa">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>DFA最小化</string>
          </property>
         </widget>
        </item>
        <item>
          <widget class="QTableWidget" name="tableWidget_minDfa">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
       <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>重新转换</string>
         </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_genCode">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>生成匹配代码</string>
        </property>
       </widget>
      </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>另存为</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
